src/Prelude.purs

```diff
@@ -1,3 +1,15 @@
+-- | `Prelude` is a module that re-exports many other foundational modules from the `purescript-prelude` library
+-- | (e.g. the Monad type class hierarchy, the Monoid type classes, Eq, Ord, etc.).
+-- |
+-- | Typically, this module will be imported in most other libraries and projects as an open import.
+-- |
+-- | ```
+-- | module MyModule where
+-- |
+-- | import Prelude -- open import
+-- |
+-- | import Data.Maybe (Maybe(..)) -- closed import
```

Comment:
I’m not sure we need to describe this as a closed import, or comment next to the Prelude import as an open one, but I don’t feel too strongly about it

Reply:
If the 'open import' comment confuses people, we can see how to edit this later. For now, I think it does more good than harm.